core/dcschema.mcf

```diff
@@ -2858,3 +2858,24 @@ Node: dcid:Portion
 typeOf: dcs:UnitOfMeasure
 name: "Portion"
 description: "A unit of measure used to measure serving of food."
+
+Node: dcid:exchangeRate
+typeOf: schema:Property
+name: "exchangeRate"
+description: "The exchange rate of a currency compared to another currency."
```

Comment:
Can we also clarify here what the base price is (along the lines of what Ajai brought up during the review). An example would be great.